src/bin/wasm2obj.rs

```diff
@@ -240,13 +253,25 @@ fn handle_module(
 
     // TODO: use the traps information
     let (compilation, relocations, address_transform, value_ranges, stack_slots, _traps) =
-        Cranelift::compile_module(
-            &module,
-            lazy_function_body_inputs,
-            &*isa,
-            generate_debug_info,
-        )
-        .map_err(|e| e.to_string())?;
+        match strategy {
+            CompilationStrategy::Auto | CompilationStrategy::Cranelift => {
+                Cranelift::compile_module(
+                    &module,
+                    lazy_function_body_inputs,
+                    &*isa,
+                    generate_debug_info,
+                )
+                .map_err(|e| e.to_string())?
+            }
+            #[cfg(feature = "lightbeam")]
+            CompilationStrategy::Lightbeam => Lightbeam::compile_module(
```

Comment:
Looks like you forgot to bring in Lightbeam. 
```
wasmtime_environ::lightbeam::Lightbeam::compile_module
or 
use wasmtime_environ::lightbeam::Lightbeam 
```
is needed.

Reply:
Oops, that's what I get for making last-minute changes and forgetting to enable --features=lightbeam when I tested it :o)